fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/derby/DerbyMaster.java

```diff
@@ -47,18 +55,28 @@
      */
     public DerbyMaster(String database) {
         this.database = database;
-                
-        try {
+
+        try (PrintWriter out = new PrintWriter(DERBY_PROPERTIES)){
```

Comment:
if I understand this correctly, the following entries are being written to the derby properties file?

Reply:
yes, this is exactly what this does.